<launch>

	<!-- Participant ID -->
	<arg name="ID" default="0"/>

	<!-- Method type ('A'- impedance, 'B'- learning, 'C'- demonstration) -->
	<arg name="method_type" default="A"/>

	<!-- Record data mode -->
	<arg name="record" default="F"/>

	<!-- Debug mode -->
	<arg name="debug" default="F"/>

	<!-- Feature learning method (ALL, MAX, BETA)-->
	<arg name="feat_method" default="ALL"/>

	<!-- List of string features that need correcting -->
	<arg name="feat_list" default="table"/>

	<!-- Location with trajectory cache -->
	<arg name="traj_cache" default="None"/>
	
	<!-- Location with trajectory random -->
	<arg name="traj_rand" default="None"/>

	<!-- Location with trajectory optimal -->
	<arg name="traj_optimal" default="None"/>

	<!-- velocity-based with trajopt planning PID -->
	<node name="pid_trajopt" pkg="beta_adaptive_pHRI" type="discrete_pid_trajopt.py"
	args="$(arg ID) $(arg method_type) $(arg record) $(arg debug)
	$(arg feat_method) $(arg feat_list) $(arg traj_cache) $(arg traj_rand) $(arg
	traj_optimal)" output="screen"/>

</launch>
